scan/src/bindings/JsBuffer.re

```diff
@@ -0,0 +1,26 @@
+type t;
+
+[@bs.val] external from: array(int) => t = "Buffer.from";
+
+[@bs.val] external _from: (string, string) => t = "Buffer.from";
+let fromHex = hexstr =>
+  if (hexstr->String.sub(0, 2) == "0x") {
+    hexstr->String.lowercase_ascii->String.sub(2, hexstr->String.length - 2)->_from("hex");
```

Comment:
Let's have `HexUtils` file and create `strip0xPrefix` function that takes a hex and removes `0x` or `0X` prefix. Then we can re-use that function everywhere.